application/src/main/java/org/togetherjava/tjbot/commands/moderation/ReportCommand.java

```diff
@@ -154,43 +155,51 @@ private boolean isAuthorOnCooldown(long userId) {
             .isPresent();
     }
 
-    private MessageCreateAction createModMessage(String modalMessage, long userID,
-            String reportedMessage, String reportedMessageUrl, TextChannel modMailAuditLog) {
-        MessageEmbed embed = new EmbedBuilder().setTitle("Report")
-            .setDescription(("""
+    private MessageCreateAction createModMessage(String reportReason, User author,
+            String reportedMessage, String reportedMessageUrl, String reportedMessageTimestamp,
+            TextChannel modMailAuditLog) {
+        MessageEmbed reportedMessageEmbed = new EmbedBuilder().setTitle("Report")
+            .setDescription("""
                     Reported Message: **%s**
-                    [Reported Message URL](%s)
-                    Reason: **%s**""").formatted(reportedMessage, reportedMessageUrl, modalMessage))
-            .setAuthor("Author ID: %d".formatted(userID))
+                    [Reported Message URL](%s)""".formatted(reportedMessage, reportedMessageUrl))
+            .setAuthor(author.getName(), null, author.getAvatarUrl())
+            .setTimestamp(Instant.parse(reportedMessageTimestamp))
             .setColor(AMBIENT_COLOR)
             .build();
 
-        return modMailAuditLog.sendMessageEmbeds(embed);
+        MessageEmbed reportReasonEmbed = new EmbedBuilder().setTitle("Reason")
+            .setDescription(reportReason)
+            .setColor(AMBIENT_COLOR)
+            .build();
+        return modMailAuditLog.sendMessageEmbeds(reportedMessageEmbed, reportReasonEmbed);
     }
 
     private void sendModMessage(ModalInteractionEvent event, List<String> args,
             TextChannel modMailAuditLog) {
-        InteractionHook hook = event.getHook();
-
         String reportedMessage = args.get(0);
         String reportedMessageUrl = args.get(1);
+        String reportedMessageTimestamp = args.get(2);
```

Comment:
parse it into Instant right here. dont keep things raw for longer than necessary.